apps/ui/src/views/Auction/Bids.vue

```diff
@@ -1 +1,180 @@
+      () => props.network,
+      () => props.auctionId,
+      'pricing'
+    ],
+    queryFn: async () => {
+      const chainId = CHAIN_IDS[props.network];
+      if (!(chainId in COINGECKO_ASSET_PLATFORMS)) {
+        return 0;
+      }
+
+      const coingeckoAssetPlatform =
+        COINGECKO_ASSET_PLATFORMS[
+          chainId as keyof typeof COINGECKO_ASSET_PLATFORMS
+        ];
+
+      const coins = await getTokenPrices(coingeckoAssetPlatform, [
+        props.auction.addressBiddingToken
+      ]);
+
+      return (
+        coins[props.auction.addressBiddingToken.toLocaleLowerCase()]?.usd ?? 0
+      );
+    }
+  });
+
+function handleEndReached() {
+  if (!hasNextPage.value) return;
+
+  fetchNextPage();
+}
+
+function getOrderPercentage(order: OrderDetailFragment) {
+  return Number(order.buyAmount) / Number(props.auction.exactOrder.sellAmount);
+}
+</script>
+
+<template>
+  <div>
+    <UiSectionHeader label="Bids" sticky />
+    <UiColumnHeader class="z-40 overflow-hidden gap-3">
+      <div class="flex-1 min-w-12 truncate">Bidder</div>
+      <div class="max-w-[168px] w-[168px] truncate">Date</div>
+      <div class="max-w-[168px] w-[168px] truncate">Amount</div>
+      <div class="max-w-[168px] w-[168px] text-right truncate">Price</div>
+    </UiColumnHeader>
+    <UiLoading
+      v-if="isPending || isBiddingTokenPriceLoading"
+      class="px-4 py-3 block"
+    />
+    <UiStateWarning v-else-if="isError" class="px-4 py-3">
+      Failed to load bids.
+    </UiStateWarning>
+    <UiStateWarning
+      v-else-if="data?.pages.flat().length === 0"
+      class="px-4 py-3"
+    >
+      There are no bids here.
+    </UiStateWarning>
+    <UiContainerInfiniteScroll
+      v-else-if="data && typeof biddingTokenPrice === 'number'"
+      :loading-more="isFetchingNextPage"
+      @end-reached="handleEndReached"
+    >
+      <template #loading>
+        <UiLoading class="px-4 py-3 block" />
+      </template>
+      <div
+        class="divide-y divide-skin-border flex flex-col justify-center border-b"
+      >
+        <div
+          v-for="order in data?.pages.flat()"
+          :key="order.id"
+          class="flex justify-between items-center gap-3 py-3 px-4 relative"
+        >
+          <div
+            class="right-0 top-0 h-[8px] absolute choice-bg opacity-20 _1"
+            :style="{
+              width: `${Math.min(getOrderPercentage(order) * 100, 100).toFixed(2)}%`
+            }"
+          />
+          <div
+            class="leading-[22px] flex-1 flex min-w-12 items-center space-x-3 truncate"
+          >
+            <UiStamp :id="order.userAddress" :size="32" />
+            <div class="flex flex-col truncate">
+              <h4
+                class="truncate"
+                v-text="order.name || shortenAddress(order.userAddress)"
+              />
+              <UiAddress
+                :address="order.userAddress"
+                class="text-[17px] text-skin-text truncate"
+              />
```

Comment:
Maybe better to redirect to profile page similar to other tables, when we click on bidder, not too important 

<img width="301" height="556" alt="Image" src="https://github.com/user-attachments/assets/770bf3e7-0d37-494b-9134-1229fa02d7da" />